autogtp/Worker.h

```diff
@@ -19,11 +19,12 @@
 #ifndef WORKER_H
 #define WORKER_H
 
+#include <QAtomicInt>
+#include <QString>
+#include <QThread>
 #include "Job.h"
 #include "Order.h"
-
-#include <QThread>
-#include <QMutex>
+#include "Result.h"
```

Comment:
I do not like to include a .h that I do not need in a .h only because all the .cpp using the second .h use also the first. I like the include to be explicit. This invalidate a lot of other changes in the autogtp code.

Reply:
Same as above.